lib/helpers/percentLength.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Convert the length to pixel unit
+ * @param length
+ * @returns {string}
+ */
+export function convertToStringCommon(length){
+    const convertUnit = 'px';
+
+    if (length == 'auto') { // tslint:disable-line
```

Comment:
we don't use tslint. replace with eslint please

Reply:
Removed this file. I added addPx function in helpers.js on pre-development branch.